runtime/etiao/src/main/java/quarks/runtime/etiao/EtiaoJob.java

```diff
@@ -51,6 +53,10 @@ public EtiaoJob(DirectGraph graph, String topologyName, ServiceContainer contain
         ControlService cs = container.getService(ControlService.class);
         if (cs != null)
             cs.registerControl(JobMXBean.TYPE, getId(), getName(), JobMXBean.class, new EtiaoJobBean(this));
+        
+        this.jobs = (JobRegistry) container.getService(JobRegistryService.class);
```

Comment:
Cannot assume the JobRegistryService is a JobRegistry which implies the add method should be on the JobRegistryService interface.